package.json

```diff
@@ -123,4 +123,4 @@
     "web3-utils": "^1.2.8",
     "yargs": "^15.3.1"
   }
-}
+}
```

Comment:
nit: missing newline at end of file.

Reply:
Can you please tell me why this keeps happening? I am obviously not intentionally removing this every time.

Reply:
How are you installing dependencies? For me it always adds a new line when installed from the commandline.

Reply:
`yarn add`

Reply:
```suggestion
}

```